backend/index.js

```diff
@@ -156,6 +157,7 @@ const controller = controllerFactory({
   csvParserService: csvParserService({ fs, isBinaryFileSync }),
   offenderService: offenderServiceFactory(elite2Api),
   offenderActivitesService: offenderActivitesFactory(elite2Api, whereaboutsApi),
+  whereaboutsDashboardService: whereaboutsDashboardFactory(elite2Api, whereaboutsApi),
```

Comment:
Don't think this is used is it? Params are wrong and you're calling `whereaboutsDashboardFactory` directly on lines 259...262